docs/RC/GetInteriorVehicleDataConsent/assets/GetInteriorVehicleDataConsent.gliffy

```diff
@@ -0,0 +1 @@
le.no_fill_no_line_v1","strokeWidth":2.0,"strokeColor":"#c27ba0","fillColor":"#FFFFFF","gradient":false,"dashStyle":null,"dropShadow":false,"state":0,"opacity":1.0,"shadowX":4.0,"shadowY":4.0}},"children":[{"x":0.0,"y":0.0,"rotation":0.0,"id":366,"width":820.0,"height":32.0,"uid":null,"order":148,"lockAspectRatio":false,"lockShape":false,"graphic":{"type":"Text","Text":{"overflow":"none","paddingTop":2,"paddingRight":2,"paddingBottom":2,"paddingLeft":2,"outerPaddingTop":6,"outerPaddingRight":6,"outerPaddingBottom":2,"outerPaddingLeft":6,"type":"fixed","lineTValue":null,"linePerpValue":null,"cardinalityType":null,"html":"<p style=\"text-align:left;\"><span style=\"font-style:italic;font-weight:bold;text-decoration:none;\">&nbsp;</span><span style=\"font-style:italic;font-weight:bold;\">&#96;</span><span style=\"font-style:italic;font-weight:bold;\"><span style=\"\">ASK_DRIVER</span><span style=\"\">&#96; -&nbsp;</span></span><span style=\"font-style:italic;font-weight:bold;\">HMI responds with error code or invalid data</span></p><p style=\"text-align:center;\"><span style=\"text-decoration:none;\">&nbsp;</span></p>","tid":null,"valign":"top","vposition":"none","hposition":"none"}},"hidden":false,"layerId":"ZZTXXXm0Cixm"}],"hidden":false,"layerId":"ZZTXXXm0Cixm"}],"hidden":false,"layerId":"ZZTXXXm0Cixm"},{"x":898.0,"y":1441.0,"rotation":0.0,"id":367,"width":100.0,"height":100.0,"uid":"com.gliffy.shape.uml.uml_v2.activity.object_flow","order":149,"lockAspectRatio":false,"lockShape":false,"graphic":{"type":"Line","Line":{"strokeWidth":1.0,"strokeColor":"#000000","fillColor":"none","dashStyle":"8.0,2.0","startArrow":0,"endArrow":1,"startArrowRotation":"auto","endArrowRotation":"auto","interpolationType":"linear","cornerRadius":10.0,"controlPath":[[2.020135363317422,-1.1368683772161603E-13],[-168.97985700728805,-1.1368683772161603E-13],[-168.97985700728805,-2.0416564941407387],[-339.9798493778935,-2.0416564941407387]],"lockSegments":{},"ortho":true}},"linkMap":[],"children":[{"x":0.0,"y":0.0,"rotation":0.0,"id":368,"width":264.0,"height":14.0,"uid":null,"order":151,"lockAspectRatio":false,"lockShape":false,"graphic":{"type":"Text","Text":{"overflow":"both","paddingTop":2,"paddingRight":2,"paddingBottom":2,"paddingLeft":2,"outerPaddingTop":6,"outerPaddingRight":6,"outerPaddingBottom":2,"outerPaddingLeft":6,"type":"fixed","lineTValue":0.49627791563275436,"linePerpValue":0.0,"cardinalityType":null,"html":"<p style=\"text-align:center;\"><span style=\"\">GetInteriorVehicleDataConcent(&#34;invalid params&#34;)</span></p>","tid":null,"valign":"middle","vposition":"none","hposition":"none"}},"hidden":false,"layerId":"ZZTXXXm0Cixm"}],"hidden":false,"layerId":"ZZTXXXm0Cixm"},{"x":502.9999999999998,"y":1506.0,"rotation":0.0,"id":371,"width":100.0,"height":100.0,"uid":"com.gliffy.shape.uml.uml_v2.activity.object_flow","order":152,"lockAspectRatio":false,"lockShape":false,"graphic":{"type":"Line","Line":{"strokeWidth":1.0,"strokeColor":"#000000","fillColor":"none
```

Comment:
Please change all instances of `GetInteriorVehicleDataConcent` and `concent` to `GetInteriorVehicleDataConsent` and `consent`

Reply:
changed in https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/171/commits/b82314d840a58d2d8326fe994b10e689acedabde